server/controllers/routes/admin/cohort/deleteCohort.js

```diff
@@ -0,0 +1,21 @@
+const { deleteCohort } = require('../../../../database/queries');
```

Comment:
@MohammedAlghazali I think these deleteCohort changes are originally from #43, the `29-DELETE-cohort` branch, maybe because you created this branch (`32-PUT-project`) branch *from* the `29-DELETE-cohort` branch.
In that case, you can make a PR into the source branch (`32-PUT-project` -> `29-DELETE-cohort`) so that other code reviewers only have to look at the PUT-related changes in github instead of seeing both sets of changes in one pull request.
Then, you could finish the DELETE pull request first, merge that into master, and then change this PR's target branch to master (so then it becomes `32-PUT-project` -> `master`).
Let me know if that makes sense or not - it's a technique you can use to keep other unrelated changes from showing up in your PRs when others are reviewing on github.

Reply:
really I needed this idea for this branch,
yes I understand this point, it's like when you make a fork and need to make push to your repo
![tom](https://user-images.githubusercontent.com/55782435/78070755-03df8600-73a5-11ea-8cd9-d2a5a59007d5.png)

Reply:
Yeah - I do that workflow a lot too, where I make a new branch from an existing branch that is still being peer-reviewed, because I need the first branch's code.
So by switching the base branch in the github PR like in your screenshot, that's how you can do that workflow for yourself, but also keep your pull requests small and focused for your peers :)